plenum/server/node.py

```diff
@@ -2221,10 +2221,10 @@ def send(self, msg: Any, *rids: Iterable[int], signer: Signer = None):
                      .format(self, msg, recipientsNum, remoteNames))
         self.nodestack.send(msg, *rids, signer=signer)
 
-    def getReplyFromLedger(self, ledger, request):
+    def getReplyFromLedger(self, ledger, request, seqNo=None):
```

Comment:
Default value of `request` should be None as you will not pass `request` when you pass `seqNo`, also use snake_case